### ISSUE TITLE
[qgroundcontrol] `libicudata.so.56` no such file

### ISSUE THREAD

**Issue by acxz** [acxz/pkgbuilds#103]
```
Post Link Common
cp: cannot stat '/usr/lib/libicudata.so.56': No such file or directory
make: *** [Makefile:2503: staging/QGroundControl] Error 1
```

upstream issue: https://github.com/mavlink/qgroundcontrol/issues/8327

**Comment by acxz** [comment]
patched with 64973d5

**Comment by acxz** [comment]
patch removed as it has been incorporated in new release: 595ca3b
